Volume/Supporting Views/VolumeMessage.swift

```diff
@@ -33,24 +41,26 @@ enum Message {
 
 struct VolumeMessage: View {
     @State var message: Message
+    @State var largeFont : Bool
+    @State var fullWidth : Bool
 
     var body: some View {
         VStack(spacing: 10) {
-            Image("volume-logo")
+            Image.volume.feed
                 .foregroundColor(.volume.orange)
             Text(message.title)
-                .font(.newYorkMedium(size: 12))
+                .font(.newYorkMedium(size: largeFont ? 24 : 18))
             Text(message.subtitle)
-                .font(.helveticaRegular(size: 10))
+                .font(.helveticaRegular(size: 12))
                 .multilineTextAlignment(.center)
                 .lineLimit(2)
         }
-        .frame(width: 205, height: 100)
+        .frame(width: fullWidth ? nil : 205, height: 100)
```

Comment:
Setting width to 0 results in no visible frame whereas setting width to nil allows it to stretch based on the view it is placed in. This is needed since we have different width messages. For example the Volume Message in the "Following - Home"  screenshot has a fixed width with multiline wrapped text but for "Bookmarks" screenshot we need a full screen width Volume Messages.

Reply:
interesting